pkg/cloud/scope/session.go

```diff
@@ -33,7 +33,13 @@ func sessionForRegion(region string) (*session.Session, error) {
 		return s.(*session.Session), nil
 	}
 
-	ns, err := session.NewSession(aws.NewConfig().WithRegion(region))
+	ns, err := session.NewSessionWithOptions(session.Options{
```

Comment:
Just noticed, are we specifically missing the addition of
`SharedConfigState: session.SharedConfigEnable` ?

Reply:
I was planning to rely on user specifying `AWS_SDK_LOAD_CONFIG`  if they need that changed from the default. https://docs.aws.amazon.com/sdk-for-go/v1/developer-guide/sessions.html#sessions-with-shared-config

Reply:
Understood